experimental/class-features.md

```diff
@@ -0,0 +1,63 @@
+
+# Class Features
+
+This experimental extension covers three class features proposals: 
+[Class Fields], [Static Class Features] and [Private Methods].
+
+## ClassBody
+
+```js
+extend interface ClassBody <: Node {
+    body: [ MethodDefinition | PropertyDefinition ];
+}
+```
+
+## PropertyDefinition
+
+```js
+interface PropertyDefinition <: Node {
+    type: "PropertyDefinition";
+    key: Expression | PrivateName;
+    value: Expression;
+    computed: boolean;
+    static: boolean;
+    private: boolean;
+}
+```
+
+- When `private` is `true`, `computed` must be `false` and `key` must be a `PrivateName`.
+- When `private` is `false`, `key` must be an `Expression` and can not be a `PrivateName`.
+
+## MethodDefinition
+
+```js
+extend interface MethodDefinition <: Node {
+    key: Expression | PrivateName;
+    static: boolean;
```

Comment:
The `static: boolean` property is also available in https://github.com/estree/estree/blob/master/es2015.md#methoddefinition

I think this is a mistake and `MethodDefinition ` in `es2015.md` should not have such a property, although acorn has supported it.

Reply:
Why do you think it's a mistake? Static methods were available in ES2015.

Reply:
Oh! My bad! I thought it is introduced in the static proposal, will update it.